tools/clang/tools/dxa/dxa.cpp

```diff
@@ -241,20 +241,21 @@ bool DxaContext::AddPart(const char *pName, const char *pPartBinFilename) {
       // Infer the output filename if needed.
       if (OutputFilename.empty()) {
         if (InputFilename == "-") {
-          OutputFilename = "-";
+          printf("Not support streaming input.\n");
```

Comment:
"Standard input/output not supported when adding a container part."

And this verification needs to happen before loading, with a check to make sure there's a defined OutputFilename as well.